<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>933</width>
    <height>715</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="imageDownloadButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Display Map</string>
    </property>
   </widget>
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>420</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid black;
border-image:url(:images/map.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="zipCodeEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>240</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="inputMask">
     <string>99999</string>
    </property>
    <property name="text">
     <string>98119</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="weatherDownloadButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>240</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Display Weather</string>
    </property>
   </widget>
   <widget class="QLabel" name="weatherLabel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>450</y>
      <width>231</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid black</string>
    </property>
    <property name="text">
     <string>Weather Information</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QStackedWidget" name="clockWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>420</width>
      <height>221</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid black</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="timeLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>261</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Time:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="worldButton">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>160</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
border-style: solid;
border-width: 1px;
border-radius:50px;
 background-color:rgb(163, 235, 255)
}
QPushButton:pressed {
    background-color: rgb(103, 103, 103)
}</string>
      </property>
      <property name="text">
       <string>World Clock</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="secondLCD">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>121</width>
        <height>111</height>
       </rect>
      </property>
      <property name="digitCount">
       <number>2</number>
      </property>
      <property name="value" stdset="0">
       <double>22.000000000000000</double>
      </property>
      <property name="intValue" stdset="0">
       <number>22</number>
      </property>
     </widget>
     <widget class="QLCDNumber" name="minuteLCD">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>121</width>
        <height>111</height>
       </rect>
      </property>
      <property name="digitCount">
       <number>2</number>
      </property>
      <property name="value" stdset="0">
       <double>22.000000000000000</double>
      </property>
      <property name="intValue" stdset="0">
       <number>22</number>
      </property>
     </widget>
     <widget class="QLCDNumber" name="hourLCD">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>131</width>
        <height>121</height>
       </rect>
      </property>
      <property name="digitCount">
       <number>2</number>
      </property>
      <property name="value" stdset="0">
       <double>22.000000000000000</double>
      </property>
      <property name="intValue" stdset="0">
       <number>22</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="currentButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
border-style: solid;
border-width: 1px;
border-radius:50px;
 background-color:rgb(163, 235, 255)
}
QPushButton:pressed {
    background-color: rgb(103, 103, 103)
}</string>
      </property>
      <property name="text">
       <string>Current Time</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="worldSecondLCD">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>30</y>
        <width>81</width>
        <height>71</height>
       </rect>
      </property>
      <property name="digitCount">
       <number>2</number>
      </property>
      <property name="value" stdset="0">
       <double>22.000000000000000</double>
      </property>
      <property name="intValue" stdset="0">
       <number>22</number>
      </property>
     </widget>
     <widget class="QLCDNumber" name="worldHourLCD">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>101</width>
        <height>91</height>
       </rect>
      </property>
      <property name="digitCount">
       <number>2</number>
      </property>
      <property name="value" stdset="0">
       <double>22.000000000000000</double>
      </property>
      <property name="intValue" stdset="0">
       <number>22</number>
      </property>
     </widget>
     <widget class="QLCDNumber" name="worldMinuteLCD">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>30</y>
        <width>81</width>
        <height>71</height>
       </rect>
      </property>
      <property name="digitCount">
       <number>2</number>
      </property>
      <property name="value" stdset="0">
       <double>22.000000000000000</double>
      </property>
      <property name="intValue" stdset="0">
       <number>22</number>
      </property>
     </widget>
     <widget class="QComboBox" name="worldComboBox">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>110</y>
        <width>281</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>New York</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Los Angeles</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Dubai</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Seoul</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ho Chi Minh</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hong Kong</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Beijing</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Tokyo</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sydney</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Paris</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>London</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="worldTimeLabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>140</y>
        <width>261</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Time:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>461</width>
      <height>431</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>150</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QLabel" name="photoLabel">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>450</y>
      <width>221</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:images/1.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>933</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionOpen_a_to_do_list"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_a_to_do_list">
   <property name="text">
    <string>Open a To-Do list</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
